### PR TITLE
[FR] Update `AGTS 2019`

### PR DESCRIPTION
**Pull request by Soetch**
I did the verification with the en.md version, and nothing is outdated.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by DoryanR** [comment]
I'm not sure how you checked the article, but there are some glaring differences:

Example here, with the table which is not the same

Code_y8T6HCNKL1

Or this more subtle formatting difference

Code_ZQoDjZzgSR

For me the article is not at all up to date, and needs further checking.

- [ ] Please don't merge while waiting for a real update of the article.

**Comment by Soetch** [comment]
Is it better now ? I see no more differences, I updated the badge and the table, with other minor changes. Sorry for making this much errors, I'll try working at another thing on this project, ahah.

**Review by TPGPL — CHANGES_REQUESTED** [review]
match results are using outdated formatting + they are missing at least 18 lines

**Review by TPGPL — APPROVED** [review]
brought formatting up-to-date

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 1]
```diff
@@ -33,7 +32,7 @@ L'**Advanced Global Taiko Showdown 2019** (***AGTS 2019***) est un tournoi par p
 | ![Couronne en argent](/wiki/shared/crown-silver.png "2ème place") | Une bannière d'équipe exclusive |
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:32]:
```suggestion
| ![Couronne d'argent](/wiki/shared/crown-silver.png "2ème place") | Une bannière d'équipe exclusive |
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 2]
```diff
@@ -33,7 +32,7 @@ L'**Advanced Global Taiko Showdown 2019** (***AGTS 2019***) est un tournoi par p
...
 | ![Couronne en bronze](/wiki/shared/crown-bronze.png "3ème place") | Une bannière d'équipe exclusive |
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:33]:
```suggestion
| ![Couronne de bronze](/wiki/shared/crown-bronze.png "3ème place") | Une bannière d'équipe exclusive |
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 3]
```diff
@@ -33,7 +32,7 @@ L'**Advanced Global Taiko Showdown 2019** (***AGTS 2019***) est un tournoi par p
...
 
-![Badge des gagnants de l'AGTS 2019](img/badge.png)
+![](img/badge.png "Badge du vainqueur de l'AGTS 2019")
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:35]:
```suggestion
![](img/badge.png "Badge des gagnants de l'AGTS 2019")
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 4]
```diff
@@ -49,13 +48,13 @@ L'Advanced Global Taiko Showdown 2019 était organisée par plusieurs membres de
...
 | Traducteur | ::{ flag=BR }:: [AnonX32](https://osu.ppy.sh/users/2730270), ::{ flag=KR }:: [BamgoeSN](https://osu.ppy.sh/users/1152851), ::{ flag=DE }:: [Capu](https://osu.ppy.sh/users/2474015), ::{ flag=HK }:: [Cynplytholowazy](https://osu.ppy.sh/users/3901754), ::{ flag=PL }:: [Deroo](https://osu.ppy.sh/users/8360475), ::{ flag=EC }:: [Gamelan4](https://osu.ppy.sh/users/9856910), ::{ flag=TW }:: [InfinityAstral](https://osu.ppy.sh/users/9913256), ::{ flag=US }:: [jyake](https://osu.ppy.sh/users/9099822), ::{ flag=FR }:: [Kasumii-sama](https://osu.ppy.sh/users/6177263), ::{ flag=FR }:: [TLQ\_Yoshii](https://osu.ppy.sh/users/7157133) |
-| Editeur du Wiki | ::{ flag=ID }:: [fajar13k](https://osu.ppy.sh/users/7100002) |
 | Développeur | ::{ flag=DE }:: [hallowatcher](https://osu.ppy.sh/users/1874761), ::{ flag=FR }:: [ThePooN](https://osu.ppy.sh/users/718454) |
+| Editeur du Wiki | ::{ flag=ID }:: [fajar13k](https://osu.ppy.sh/users/7100002) |
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:52]:
```suggestion
| Éditeur du Wiki | ::{ flag=ID }:: [fajar13k](https://osu.ppy.sh/users/7100002) |
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 5]
```diff
@@ -49,13 +48,13 @@ L'Advanced Global Taiko Showdown 2019 était organisée par plusieurs membres de
...
 
 ## Liens
 
 - [Thread de discussion](https://osu.ppy.sh/community/forums/topics/931003)
-- [Serveur Discord de l'AGTS](https://discord.gg/zbHVzvF)
+- [Serveur Discord de l'GTS](https://discord.gg/3mGC3HB)
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:57]:
```suggestion
- [Serveur Discord du GTS](https://discord.gg/3mGC3HB)
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 6]
```diff
@@ -112,9 +111,9 @@ Cette compétition est maintenant terminée, voici le classement final:
 
 | Classement | Pays |
 | :-: | :-- |
-| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=JP }:: Japon |
-| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=CA }:: Canada |
-| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=RU }:: Russie A |
+| ![Couronne d'or](/wiki/shared/crown-gold.png "1ère place") | ::{ flag=JP }:: Japon |
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:114]:
```suggestion
| ![Couronne en or](/wiki/shared/crown-gold.png "1ère place") | ::{ flag=JP }:: Japon |
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 7]
```diff
@@ -283,38 +282,50 @@ Cette compétition est maintenant terminée, voici le classement final:
 
 ### Grande Finale
 
-| Samedi, 14/09/2019 |  |  |  |  |
+Samedi, 14 Septembre 2019:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:285]:
```suggestion
Samedi 14 septembre 2019 :
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 8]
```diff
@@ -283,38 +282,50 @@ Cette compétition est maintenant terminée, voici le classement final:
...
+
+| Équipe 1 |  |  | Équipe 2 | Lien du match |
 | --: | :-: | :-: | :-- | :-- |
 | **Japon** ::{ flag=JP }:: | **7** | 1 | ::{ flag=CA }:: Canada | [#1](https://osu.ppy.sh/community/matches/54839968) |
 
 ### Finales
 
-| Samedi, 07/09/2019 |  |  |  |  |
+Samedi, 7 Septembre 2019:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:293]:
```suggestion
Samedi 7 septembre 2019 :
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 9]
```diff
@@ -283,38 +282,50 @@ Cette compétition est maintenant terminée, voici le classement final:
...
+
+| Équipe 1 |  |  | Équipe 2 | Lien du match |
 | --: | :-: | :-: | :-- | :-- |
 | **Japon** ::{ flag=JP }:: | **7** | 1 | ::{ flag=RU }:: Russie A | [#1](https://osu.ppy.sh/community/matches/54674122) |
 
-| Dimanche, 08/09/2019 |  |  |  |  |
+Dimanche, 8 Septembre 2019:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:299]:
```suggestion
Dimanche 8 septembre 2019 :
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 10]
```diff
@@ -283,38 +282,50 @@ Cette compétition est maintenant terminée, voici le classement final:
...
+
+| Équipe 1 |  |  | Équipe 2 | Lien du match |
 | --: | :-: | :-: | :-- | :-- |
 | Corée du Sud ::{ flag=KR }:: | 4 | **7** | ::{ flag=CA }:: **Canada** | [#1](https://osu.ppy.sh/community/matches/54691282) |
 | Russie A ::{ flag=RU }:: | 2 | **7** | ::{ flag=CA }:: **Canada** | [#1](https://osu.ppy.sh/community/matches/54706574) |
 
 ### Demi-finales
 
-| Samedi, 31/08/2019 |  |  |  |  |
+Samedi, 31 Aout 2019:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:308]:
```suggestion
Samedi 31 août 2019 :
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 11]
```diff
@@ -283,38 +282,50 @@ Cette compétition est maintenant terminée, voici le classement final:
...
+
+| Équipe 1 |  |  | Équipe 2 | Lien du match |
 | --: | :-: | :-: | :-- | :-- |
 | Indonésie A ::{ flag=ID }:: | 4 | **7** | ::{ flag=JP }:: **Japon** | [#1](https://osu.ppy.sh/community/matches/54504410) |
 | **Russie A** ::{ flag=RU }:: | **7** | 2 | ::{ flag=TW }:: Taïwan | [#1](https://osu.ppy.sh/community/matches/54507108) |
 | Argentine ::{ flag=AR }:: | 4 | **7** | ::{ flag=CA }:: **Canada** | [#1](https://osu.ppy.sh/community/matches/54515229) |
 
-| Dimanche, 01/09/2019 |  |  |  |  |
+Dimanche, 1 Septembre 2019:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:316]:
```suggestion
Dimanche 1 septembre 2019 :
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 12]
```diff
@@ -283,38 +282,50 @@ Cette compétition est maintenant terminée, voici le classement final:
...
+| Équipe 1 |  |  | Équipe 2 | Lien du match |
 | --: | :-: | :-: | :-- | :-- |
 | Taïwan ::{ flag=TW }:: | 1 | **7** | ::{ flag=CA }:: **Canada** | [#1](https://osu.ppy.sh/community/matches/54521624) |
 | Pays-Bas ::{ flag=NL }:: | 6 | **7** | ::{ flag=KR }:: **Corée du Sud** | [#1](https://osu.ppy.sh/community/matches/54530786) |
 | Indonésie A ::{ flag=ID }:: | 3 | **7** | ::{ flag=KR }:: **Corée du Sud** | [#1](https://osu.ppy.sh/community/matches/54533409) |
 
 ### Quarts de finale
 
-| Samedi, 24/08/2019 |  |  |  |  |
+Samedi, 24 Aout 2019:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:326]:
```suggestion
Samedi 24 août 2019 :
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 13]
```diff
@@ -324,7 +335,9 @@ Cette compétition est maintenant terminée, voici le classement final:
 | Pologne ::{ flag=PL }:: | 1 | **6** | ::{ flag=ID }:: **Indonésie A** | [#1](https://osu.ppy.sh/community/matches/54320532) |
 | **Canada** ::{ flag=CA }:: | **6** | 1 | ::{ flag=RU }:: Russie B | [#1](https://osu.ppy.sh/community/matches/54322599) |
 
-| Dimanche, 25/08/2019 |  |  |  |  |
+Dimanche, 25 Aout 2019:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:338]:
```suggestion
Dimanche 25 août 2019 : 
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 14]
```diff
@@ -334,7 +347,9 @@ Cette compétition est maintenant terminée, voici le classement final:
 
 ### Phases de poules
 
-| Groupe | Samedi, 17/08/2019 |  |  |  |  |
+Samedi, 17 Aout 2019:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:350]:
```suggestion
Samedi 17 août 2019 :
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 15]
```diff
@@ -351,29 +366,31 @@ Cette compétition est maintenant terminée, voici le classement final:
 | B5 | Chili ::{ flag=CL }:: | 0 | **5** | ::{ flag=AR }:: **Argentine** | [#1](https://osu.ppy.sh/community/matches/54153575) |
 | C3 | **Canada** ::{ flag=CA }:: | **5** | 0 | ::{ flag=GB }:: ::{ flag=PT }:: Royaume-Uni/Portugal | [#1](https://osu.ppy.sh/community/matches/54153517) |
 
-| Groupe | Dimanche, 18/08/2019 |  |  |  |  |
+Dimanche, 18 Aout 2019:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:369]:
```suggestion
Dimanche 18 août 2019 : 
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 16]
```diff
@@ -49,13 +48,13 @@ L'Advanced Global Taiko Showdown 2019 était organisée par plusieurs membres de
...
 | Traducteur | ::{ flag=BR }:: [AnonX32](https://osu.ppy.sh/users/2730270), ::{ flag=KR }:: [BamgoeSN](https://osu.ppy.sh/users/1152851), ::{ flag=DE }:: [Capu](https://osu.ppy.sh/users/2474015), ::{ flag=HK }:: [Cynplytholowazy](https://osu.ppy.sh/users/3901754), ::{ flag=PL }:: [Deroo](https://osu.ppy.sh/users/8360475), ::{ flag=EC }:: [Gamelan4](https://osu.ppy.sh/users/9856910), ::{ flag=TW }:: [InfinityAstral](https://osu.ppy.sh/users/9913256), ::{ flag=US }:: [jyake](https://osu.ppy.sh/users/9099822), ::{ flag=FR }:: [Kasumii-sama](https://osu.ppy.sh/users/6177263), ::{ flag=FR }:: [TLQ\_Yoshii](https://osu.ppy.sh/users/7157133) |
-| Editeur du Wiki | ::{ flag=ID }:: [fajar13k](https://osu.ppy.sh/users/7100002) |
 | Développeur | ::{ flag=DE }:: [hallowatcher](https://osu.ppy.sh/users/1874761), ::{ flag=FR }:: [ThePooN](https://osu.ppy.sh/users/718454) |
+| Editeur du Wiki | ::{ flag=ID }:: [fajar13k](https://osu.ppy.sh/users/7100002) |
 
 ## Liens
 
 - [Thread de discussion](https://osu.ppy.sh/community/forums/topics/931003)
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:56]:
```suggestion
- [Fil de discussion](https://osu.ppy.sh/community/forums/topics/931003)
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 17]
```diff
@@ -112,9 +111,9 @@ Cette compétition est maintenant terminée, voici le classement final:
...
 | Classement | Pays |
 | :-: | :-- |
-| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=JP }:: Japon |
-| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=CA }:: Canada |
-| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=RU }:: Russie A |
+| ![Couronne d'or](/wiki/shared/crown-gold.png "1ère place") | ::{ flag=JP }:: Japon |
+| ![Couronne d'argent](/wiki/shared/crown-silver.png "2nde place") | ::{ flag=CA }:: Canada |
+| ![Couronne de bronze](/wiki/shared/crown-bronze.png "3ème place") | ::{ flag=RU }:: Russie A |
 
 ## Liste des beatmaps
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:118]:
```suggestion
## Mappools
```

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 18]
```diff
@@ -112,9 +111,9 @@ Cette compétition est maintenant terminée, voici le classement final:
 
 | Classement | Pays |
 | :-: | :-- |
-| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=JP }:: Japon |
-| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=CA }:: Canada |
-| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=RU }:: Russie A |
+| ![Couronne d'or](/wiki/shared/crown-gold.png "1ère place") | ::{ flag=JP }:: Japon |
+| ![Couronne d'argent](/wiki/shared/crown-silver.png "2nde place") | ::{ flag=CA }:: Canada |
```
Comment (Ganondorf-1) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:115]:
```suggestion
| ![Couronne d'argent](/wiki/shared/crown-silver.png "2ème place") | ::{ flag=CA }:: Canada |
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by Ganondorf-1 — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**